custom/icds_reports/management/commands/remove_duplicates.py

```diff
@@ -0,0 +1,107 @@
+from datetime import date
+
+import time
+from django.core.management.base import BaseCommand
+
+from django.db import connections, transaction
+from corehq.apps.userreports.util import get_table_name
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+from corehq.apps.userreports.tasks import _get_config_by_id, _build_indicators
+from corehq.apps.change_feed.data_sources import get_document_store_for_doc_type
+from pillowtop.dao.couch import ID_CHUNK_SIZE
+
+@transaction.atomic
+def _run_custom_sql_script(command):
+    db_alias = get_icds_ucr_citus_db_alias()
+    if not db_alias:
+        return
+
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command)
+
+    return cursor.fetchall()
+
+class Command(BaseCommand):
+    DOMAIN_NAME = 'icds-dashboard-qa'
+    UCR_NAMES = [
```

Comment:
do you mean `UCR_DATA_SOURCES`?

Also can we check all instead of having a hard coded list?

Reply:
Hey, this list is taken from the place which considers only these many UCRs in the aggregation https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/location_reassignment.py#L74

Reply:
unfortunate hard coding of things.. 
Not sure if in the scope of this PR or not, but if you can extract that as a constant, at least that would ensure new changes are available in this command as well.

Reply:
I think that might not be super easy as that variable is defined in the function. if it was in the class, it would have been easier. So, I am going to leave it as is for now.

Reply:
why that would not be easy? if we extract that as a constant and use that instead? and import that here as well.
(fine to punt this)